src/SamplesApp/SamplesApp.UITests/UnoSamples_Tests.Keyboard.cs

```diff
@@ -39,76 +39,100 @@ public void Keyboard_Textbox_InsideScrollViewer_Validation()
 				_app.TapCoordinates(0f, 0f);
+				_app.DismissKeyboard();
 				_app.Wait(1);
-				TakeScreenshot("0 - Remove Focus on normalTextBox", ignoreInSnapshotCompare: AppInitializer.GetLocalPlatform() == Platform.Android /*Keyboard predicted text can change*/);
+				//TakeScreenshot("0 - Remove Focus on normalTextBox", ignoreInSnapshotCompare: AppInitializer.GetLocalPlatform() == Platform.Android /*Keyboard predicted text can change*/);
+				var screenshot2 = TakeScreenshot("0 - Remove Focus on normalTextBox", ignoreInSnapshotCompare: true);
+				ImageAssert.AreNotEqual(screenshot1, screenshot2);
+				ImageAssert.AreNotEqual(initialScreenshot, screenshot2);
 			}
 
 			{
 				// Setting focus on normalTextBox
 				_app.FastTap(filledTextBox);
 				_app.Wait(1);
-				TakeScreenshot("1 - Focus on filledTextBox", ignoreInSnapshotCompare: true);
+				var screenshot1 = TakeScreenshot("1 - Focus on filledTextBox", ignoreInSnapshotCompare: true);
+				ImageAssert.AreNotEqual(initialScreenshot, screenshot1);
 
 				// Removing focus on normalTextBox
 				_app.TapCoordinates(0f, 0f);
+				_app.DismissKeyboard();
 				_app.Wait(1);
-				TakeScreenshot("1 - Remove Focus on filledTextBox", ignoreInSnapshotCompare: AppInitializer.GetLocalPlatform() == Platform.Android /*Keyboard predicted text can change*/);
+				var screenshot2 = TakeScreenshot("1 - Remove Focus on filledTextBox", ignoreInSnapshotCompare: true /*Keyboard predicted text can change*/);
+				ImageAssert.AreNotEqual(screenshot1, screenshot2);
+				ImageAssert.AreNotEqual(initialScreenshot, screenshot2);
 			}
 
 			{
 				// Setting focus on placeholderTextTextBox
 				_app.FastTap(placeholderTextTextBox);
 				_app.Wait(1);
-				TakeScreenshot("2 - Focus on placeholderTextTextBox", ignoreInSnapshotCompare: true);
+				var screenshot1 = TakeScreenshot("2 - Focus on placeholderTextTextBox", ignoreInSnapshotCompare: true);
+				ImageAssert.AreNotEqual(initialScreenshot, screenshot1);
 
 				// Removing focus on placeholderTextTextBox
 				_app.TapCoordinates(0f, 0f);
+				_app.DismissKeyboard();
 				_app.Wait(1);
-				TakeScreenshot("2 - Remove Focus on placeholderTextTextBox");
+				var screenshot2 = TakeScreenshot("2 - Remove Focus on placeholderTextTextBox", ignoreInSnapshotCompare: true);
+				ImageAssert.AreNotEqual(screenshot1, screenshot2);
+				ImageAssert.AreNotEqual(initialScreenshot, screenshot2);
 			}
 
 			{
 				// Setting focus on disabledTextBox
 				_app.FastTap(disabledTextBox);
 				_app.Wait(1);
-				TakeScreenshot("3 - Focus on disabledTextBox", ignoreInSnapshotCompare: true);
-
+				var screenshot1 = TakeScreenshot("3 - Focus on disabledTextBox", ignoreInSnapshotCompare: true);
+				
 				// Removing focus on disabledTextBox
 				_app.TapCoordinates(0f, 0f);
+				_app.DismissKeyboard();
 				_app.Wait(1);
-				TakeScreenshot("3 - Remove Focus on disabledTextBox");
+				var screenshot2 = TakeScreenshot("3 - Remove Focus on disabledTextBox");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'screenshot2'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4978073)

Reply:
i just want a final list of snapshot errors. After that i will add more in this. that's why i told you to ignore this PR for now. I am just investigation the cause and solution for error.